x/did/client/cli/tx.go

```diff
@@ -42,48 +37,29 @@ func GetCmdCreateDID(cdc *codec.Codec) *cobra.Command {
 			txBldr := authtxb.NewTxBuilderFromCLI().WithTxEncoder(utils.GetTxEncoder(cdc))
 			cliCtx := context.NewCLIContext().WithCodec(cdc).WithAccountDecoder(cdc)
 
-			privKey, err := genPrivKey(viper.GetBool(flagInteractive))
-			if err != nil {
-				return err
-			}
-
 			networkID, err := types.NewNetworkID(args[0])
 			if err != nil {
 				return err
 			}
 
-			pubKey := privKey.PubKey()
-			did := types.NewDID(networkID, pubKey, types.ES256K)
-			keyID := types.NewKeyID(did, "key1")
-			doc := types.NewDIDDocument(did, types.NewPubKey(keyID, types.ES256K, pubKey))
-
-			sig, err := types.Sign(doc, types.InitialSequence, privKey)
```

Comment:
Moved these DIDDoc gen logic to the new `newMsgCreateDID` function, because Doc generation would be complicated in the future.